files/squid_whitelist/web_whitelist

```diff
@@ -66,6 +66,7 @@ k8s.gcr.io
 keyservice.opensciencedatacloud.org
 ks.osdc.io
 lib.stat.cmu.edu
+login.bionimbus.org/Shibboleth.sso/DiscoFeed
```

Comment:
Squid whitelists domain and IP addresses, there is no need to use URL, won't work.

Correct and push before merging please

Reply:
so will adding just `login.bionimbus.org` work?

Reply:
yes